src/components/Sum.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Sum = ({
+  a,
+  b,
+}) => (
+  <p>
+    <span>{`Sum of ${a}`}</span>
+    <span>{` and ${b}`}</span>
+    <span>{` is ${a + b}`}</span>
```

Comment:
why do you even need 3 spans?
raw string literal would do just fine
```suggestion
    {`Sum of ${a} and ${b} is ${a + b}`}
```